pkg/utils/utils.go

```diff
@@ -39,3 +44,33 @@ func Contains(s []string, str string) bool {
 	}
 	return false
 }
+
+func SetEnv(envName string) error {
+	configPath := path.Join(app.WorkDir.Location, app.WorkDir.ConfigFile)
+	data, err := file.Read(configPath)
+	if err != nil {
+		return err
+	}
+	result := ""
+	r, _ := regexp.Compile(`(?:envName: [a-zA-Z]+-\w+)`)
+	match := r.FindString(string(data))
+	if match != "" {
+		result = strings.Replace(string(data), match, fmt.Sprintf("envName: %s", envName), 1)
+	} else {
+		result = string(data) + fmt.Sprintf("envName: %s\n", envName)
+	}
+	err = file.Write(configPath, result, 0755)
+	if err != nil {
+		return err
+	}
+	return err
+}
+
+func FetchEnv(envName string) string {
```

Comment:
if we are fetching the env then why would we provide envName as the argument to the function?